variants.go

```diff
@@ -13,6 +37,14 @@ type Variant struct {
 	SKU string
 	// Title is the title of the product variant.
 	Title string
+	// Option1 is a custom properties that a shop owner uses to define product variants. This is option1 of three available.
+	Option1 string
```

Comment:
Do you have an example value this might be? It it just `100ml`?

Reply:
Yeh, it's the option value that we give our variants. Here is an example from one of the products. Notice the title matches option1. If option2 and/or 3 were set it would concatenate the title with all three options with /.

```json
"variants": [
	{
	"id": 43139377463526,
	"product_id": 7886066024678,
	"title": "25mg",
	"price": "25.00",
	"sku": "3265",
	"position": 1,
	"inventory_policy": "continue",
	"compare_at_price": null,
	"fulfillment_service": "manual",
	"inventory_management": "shopify",
	"option1": "25mg",
	"option2": null,
	"option3": null,
	"created_at": "2022-07-20T13:53:49+01:00",
	"updated_at": "2022-08-18T12:12:31+01:00",
	"taxable": true,
	"barcode": "",
	"grams": 0,
	"image_id": null,
	"weight": 0,
	"weight_unit": "kg",
	"inventory_item_id": 45228185551078,
	"inventory_quantity": 999,
	"old_inventory_quantity": 999,
	"requires_shipping": true,
	"admin_graphql_api_id": "gid://shopify/ProductVariant/43139377463526"
	},
]
```